From f94e3456a3a14cb92205675efa26b6e854c93235 Mon Sep 17 00:00:00 2001
From: Xianzhong <b07117@freescale.com>
Date: Tue, 21 May 2013 02:32:08 +0800
Subject: [PATCH 200/602] ENGR00263121 enable vg acceleration for android dual
 display

vg acceleration feature is enabled when hwc.stretch.filter is set to 0

Signed-off-by: Xianzhong <b07117@freescale.com>
---
 hdmidongle_6dq/init.rc | 2 +-
 sabreauto_6q/init.rc   | 2 +-
 sabresd_6dq/init.rc    | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/hdmidongle_6dq/init.rc b/hdmidongle_6dq/init.rc
index a52334a..801438a 100755
--- a/hdmidongle_6dq/init.rc
+++ b/hdmidongle_6dq/init.rc
@@ -33,7 +33,7 @@ on boot
     setprop debug.sf.enable_hgl 1
     setprop debug.egl.hw	1
 
-    setprop hwc.stretch.filter  1
+    setprop hwc.stretch.filter  0
     setprop hwc.enable_dither   1
 
     # fsl omx graphic manager media framework property
diff --git a/sabreauto_6q/init.rc b/sabreauto_6q/init.rc
index da26f1b..4227af9 100644
--- a/sabreauto_6q/init.rc
+++ b/sabreauto_6q/init.rc
@@ -33,7 +33,7 @@ on boot
     setprop debug.sf.enable_hgl 1
     setprop debug.egl.hw	1
 
-    setprop hwc.stretch.filter  1
+    setprop hwc.stretch.filter  0
     setprop hwc.enable_dither   1
 
     # fsl omx graphic manager media framework property
diff --git a/sabresd_6dq/init.rc b/sabresd_6dq/init.rc
index 5f105d1..fd9d1e0 100755
--- a/sabresd_6dq/init.rc
+++ b/sabresd_6dq/init.rc
@@ -38,7 +38,7 @@ on boot
     setprop debug.sf.enable_hgl 1
     setprop debug.egl.hw	1
 
-    setprop hwc.stretch.filter  1
+    setprop hwc.stretch.filter  0
     setprop hwc.enable_dither   1
 
     # fsl omx graphic manager media framework property
-- 
1.8.0

